siliconcompiler/apps/sc_remote.py

```diff
@@ -129,13 +129,14 @@ def main():
     # If the -reconnect flag is specified, re-enter the client flow
     # in its "check_progress/ until job is done" loop.
     elif args['reconnect']:
-        # Remove entry steps from the steplist, so that they are not fetched from the remote.
-        remote_steps = chip.list_steps()
+        # Start from succesors of entry nodes, so entry nodes are not fetched from remote.
         environment = copy.deepcopy(os.environ)
         entry_nodes = chip._get_flowgraph_entry_nodes(chip.get('option', 'flow'))
-        for node in entry_nodes:
-            remote_steps.remove(node[0])
-        chip.set('option', 'steplist', remote_steps)
+        flow = chip.get('option', 'flow')
+        entry_nodes = chip._get_flowgraph_entry_nodes(flow)
+        for entry_node in entry_nodes:
+            outputs = chip._get_flowgraph_node_outputs(flow, entry_node)
+            chip.set('option', 'from', list(map(lambda node: node[0], outputs)))
```

Comment:
Yes that should be handled correctly. Internally [when checking the flowgraph entry nodes](https://github.com/siliconcompiler/siliconcompiler/blob/from-to-rebase/siliconcompiler/core.py/#L4729-L4730) we just get the nodes that are [in these from steps](https://github.com/siliconcompiler/siliconcompiler/blob/from-to-rebase/siliconcompiler/core.py/#L4714-L4716). So if there are duplicates in the from steps it doesn't make a difference. (Maybe it would be a tiny bit slower if we had huge graphs)